### PR TITLE
feat-alert

### PR DESCRIPTION
**Pull request by iamsaurav28**
Alert (2)

### CODE REVIEW

File: Comp-layouts/Alert.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  
+      <div class="alert alert-dark">
+         
+          <div class="alert-text">
+            Danger! Indicate Negative Action
+          </div>
+      </div>
+  
+ 
+
```
Comment (rahulcs754) [inline Comp-layouts/Alert.html:31]: Please don't leave space

**Review by rahulcs754 — COMMENTED** [review]
Look good
You created lots of example for an alert  that is good

File: Comp-layouts/Alert.html (hunk 1 above)
Comment (iamsaurav28) [reply Comp-layouts/Alert.html:31]: okay i ll take care of it.

File: Stylesheets/Abstract/color.css [hunk 2]
```diff
@@ -1,10 +1,11 @@
 :root {
+  --white: white;
   --bg-color: #f8f8ff;
   --bg-white: #fffafa;
   --bg-gray-dark: #495057;
   --gray: #6c757d;
   --gray-light: #adb5bd;
-  --black: #050505;
+  --bg-black: #050505;
   --oc-primary: #0096c7;
```
Comment (tanishq20) [inline Stylesheets/Abstract/color.css:9]: oc stand for?
Try to give names that are easily understandable.

**Review by tanishq20 — COMMENTED** [review]
Everything looks good.
Just small clarification is needed.

File: Stylesheets/Abstract/color.css (hunk 2 above)
Comment (iamsaurav28) [reply Stylesheets/Abstract/color.css:9]: oc is open color. i ll change and give easy name :)

**Pull request merged**